content/en/modules/nibabel/index.md

```diff
@@ -0,0 +1,136 @@
+
+The original presentation and the practical notebook can be found [here](https://github.com/effigies/nibabel-presentations).
+In this exercise, we will use the notebook in the brainhack school repository. Follow the instructions below to run both. 
+
+### Downlad data and jupyter notebook
+
+You will need to install the example dataset with [DataLad](/modules/datalad).
+
+```bash
+# download open neuro dataset
+mkdir ~/brainhackschool_nibabel
+cd ~/brainhackschool_nibabel
+mkdir data
+cd data
+datalad install ///openneuro
+cd openneuro
+datalad get -r ds000114/sub-01
+
+cd ~/brainhackschool_nibabel/data
+datalad install git@github.com:OpenNeuroDerivatives/ds000228-fmriprep.git
+cd ds000228-fmriprep
+datalad get -r sourcedata/freesurfer/sub-pixar001/surf
+datalad get -r sourcedata/freesurfer/fsaverage5
+
+datalad get sub-pixar001/anat/sub-pixar001_hemi-L_pial.surf.gii
+datalad get sub-pixar001/func/sub-pixar001_task-pixar_hemi-L_space-fsaverage5_bold.func.gii
+datalad get sub-pixar001/func/sub-pixar001_task-pixar_space-fsLR_den-91k_bold.dtseries.nii
+
+# get the surface data from midnight scan club
+cd ~/brainhackschool_nibabel/data
+wget -c https://raw.githubusercontent.com/MidnightScanClub/MSCcodebase/master/Utilities/Conte69_atlas-v2.LR.32k_fs_LR.wb/Conte69.L.inflated.32k_fs_LR.surf.gii 
+
+# get the tutorial
+cd ~/brainhackschool_nibabel
+wget -c https://raw.githubusercontent.com/school-brainhack/school-brainhack.github.io/tree/main/content/en/modules/nibabel/Pipfile
+wget -c https://raw.githubusercontent.com/school-brainhack/school-brainhack.github.io/tree/main/content/en/modules/nibabel/Nibabel.py
+
+# make a temporary directory to dump things
+mkdir ~/brainhackschool_nibabel/data/tmp
+```
+
+### Set up environment
+
+You will need to create the environment with the `Pipfile`.
+
+```
+pip install pipenv  # You need pipenv to create an environmnet
+cd ~/brainhackschool_nibabel/
+pipenv install Pipfile  # install the environment
+pipenv run jupyter-notebook
+```
+From here you can navigate to the presentaion `Nibabel.py`
+
+The first comment by Hao-Ting Wang includes time stamps of this video. 
+The time stamps highlight the advance and optional sections if you wish to skip.
+You can skip the irrelevant sections.
+You need to watch all the basic content to complete the excercise.
+
+ - **Basic** - Be able to load and save different types of neuroimaging files in Nibabel
+ - **Basic** - How to work with volumetric data
+ - **Advanced** - Understand the difference between array and array proxy images
+ - **optional** - Acquire a passing familiarity with the structures of surface images, CIFTI-2 files and tractogram
+
+The video presentation is available below:
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=Y6ulmOlW1FI" title="NiBabel: Neuroimaging data and file structures in Python" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
When I built the website with hugo the video did not load ('www.youtube.com refused to connect')

Reply:
the URL should include `embed` fixing...